topic_manager.go

```diff
@@ -89,7 +89,7 @@ func checkBroker(broker Broker, config *sarama.Config) error {
 	}
 
 	err := broker.Open(config)
-	if err != nil {
+	if err != nil && !errors.Is(err, sarama.ErrAlreadyConnected) {
```

Comment:
accordin to docs, `Open` might return this if it's already connected and it's not an error.